apis/placement/v1beta1/clusterresourceplacement_types.go

```diff
@@ -438,6 +438,12 @@ const (
 	// - "False" means some of them have failed.
 	// - "Unknown" means we haven't started the apply yet.
 	ResourcesAppliedConditionType ResourcePlacementConditionType = "ResourceApplied"
+
+	// PlacementScheduledConditionType indicates whether we have successfully scheduled the placement.
+	// Its condition status can be one of the following:
+	// - "True" means we have successfully scheduled the resources to fully satisfy the placement requirement.
+	// - "False" means we didn't fully satisfy the placement requirement. We will fill the Message field.
+	PlacementScheduledConditionType ResourcePlacementConditionType = "PlacementScheduled"
```

Comment:
Isn't there already a `ClusterResourcePlacementScheduledConditionType`?

Reply:
I also had the same confusion before and misunderstood the purpose of this fields.

```
	// PlacementStatuses contains a list of placement status on the clusters that are selected by PlacementPolicy.
	// Each selected cluster according to the latest resource placement is guaranteed to have a corresponding placementStatuses.
	// In the pickN case, there are N placement statuses where N = NumberOfClusters. Some of them may not have assigned
	// clusters when we cannot fill the required number of clusters.
	// +optional
	PlacementStatuses []ResourcePlacementStatus
```

We want to use this array to explain to user why the pickN cannot be satisfied and why the remaining replicas cannot be scheduled, so that the clusterName is optional.
when clusterName is empty, we will use the condition to explain why.